protocol/protobuf/community_update.proto

```diff
@@ -64,14 +64,25 @@ message ChannelData {
   CommunityChat channel = 4;
 }
 
+message SignedCommunityEvent {
+  // Signature of the payload field
+  bytes signature = 1;
+  // Marshaled CommunityEvent
+  bytes payload = 2;
+}
+
 // CommunityEventsMessage is a message used to propagate information
 // about community changes.
 message CommunityEventsMessage {
   bytes community_id = 1;
-  
+
   // Events base CommunityDescription with owner signature on top of which events were generated
   bytes events_base_community_description = 2;
-  
+
   // A list of admins events for the channel in bytes
-  repeated bytes events = 3;
+  // Deprecated: use signed_events instead.
+  repeated bytes events = 3 [deprecated=true];
```

Comment:
> If you [update](https://protobuf.dev/programming-guides/proto3/#updating) a message type by entirely deleting a field, or commenting it out, future developers can reuse the field number when making their own updates to the type. This can cause severe issues, as described in [Consequences of Reusing Field Numbers](https://protobuf.dev/programming-guides/proto3/#consequences).

https://protobuf.dev/programming-guides/proto3/#fieldreserved

Long story short, removing fields from protobufs is not safe. Anyway, I think I will go with `reserved` approach rather than with `[deprecated=true]`.

Reply:
> Non-required fields can be removed, as long as the field number is not used again in your updated message type. You may want to rename the field instead, perhaps adding the prefix “OBSOLETE_”, or make the field number [reserved](https://protobuf.dev/programming-guides/proto2/#fieldreserved), so that future users of your .proto can’t accidentally reuse the number.

More formal way (and better IMO) of adding "OBSOLETE_" is `[deprecated=true]`.

Reply:
hm... I was changing it before...

Reply:
Changing the name is ok as long as type and semantics stays the same. Removing a field and reusing its number for something different will most likely lead to issues. Maybe you were lucky because you changed message that nobody uses.

Once you realize that older serialized messages may still contain the deleted field then you will understand why deleting fields in protobufs is not safe.